web/php/DB/Page.php

```diff
@@ -161,20 +161,7 @@ public function getSite()
 
     public function getTags()
     {
-        $c = new Criteria();
-        $c->add('page_id', $this->getPageId());
-        $tags = PageTagPeer::instance()->select($c);
-        return $tags;
-    }
-
-    public function getTagsAsArray()
```

Comment:
Can you confirm `getTagsAsArray()` does not appear at all in the codebase now?

Reply:
Fixed one that I missed, thank you. Should be entirely gone now.